saw-central/src/SAWCentral/Crucible/LLVM/Builtins.hs

```diff
@@ -726,12 +725,15 @@ refineMethodSpec cc methodSpec lemmas tactic =
                   ]
           (x:xs) -> return (x NE.:| xs)
 
-     printOutLnTop Info $
-       unwords ["Refining specification for", (methodSpec ^. csName) , "..."]
+     printOutLnTop Info $ Text.unpack $
+         "Refining specification for " <> (methodSpec ^. csName) <> "..."
 
-     unless (null irrelevantLemmas) $
-         printOutLnTop Warn $ unlines $
-           [ "Irrelevant overrides included in specification refinement for " ++ show (pretty fnm) ] ++
+     unless (null irrelevantLemmas) $ do
+         printOutLnTop Warn $ Text.unpack $
+             "Irrelevant overrides included in specification refinement for " <>
+             -- XXX do we get anything useful by text -> ppdoc -> string -> text?
+             Text.pack (show $ pretty fnm)
```

Comment:
>  do we get anything useful by text -> ppdoc -> string -> text?

I don't think so. About the only post-processing that the `Pretty Text` instance does is to convert newlines to the [`line`](https://hackage.haskell.org/package/prettyprinter-1.7.1/docs/Prettyprinter.html#v:line) combinator, but I don't think `fnm` is going to contain newlines most of the time (if ever). You'd probably be better served by just using `fnm` directly here instead of `Text.pack (show $ pretty fnm)`.